mwdb/web/src/components/Shares.js

```diff
@@ -6,19 +6,29 @@ import { RefString, DateString, ObjectLink, ConfirmationModal } from "@mwdb-web/
 
 class ShareItem extends Component {
     render() {
-        const fieldStyle = {
+        let fieldStyle = {
             wordBreak: 'break-all'
         };
+        let current_object = this.props.access_reason.includes(this.props.id)
+        let uploader = this.props.access_reason.includes(this.props.group_name)
 
-        return (<tr>
-            <td><ObjectLink type="group" id={this.props.group_name}/></td>
-            <td style={fieldStyle}><RefString
-                reason_type={this.props.reason_type}
-                related_object_dhash={this.props.related_object_dhash}
-                related_object_type={this.props.related_object_type}
-                related_user_login={this.props.related_user_login} /></td>
-            <td><DateString date={this.props.access_time}/></td>
-        </tr>)
+        if (!current_object) fieldStyle.background = 'LightGray';
```

Comment:
Uh...
1. Use `backgroundColor` instead of `background`. `background` can be used if you want to set various background properties in the same line, it works because `background-color` is the first field, but there is no reason to use it that way here.
2. Colors should not be capitilized, they're lowercase in standard: https://www.w3.org/TR/css-color-3/#valuea-def-color. Don't use w3schools :D
```suggestion
        if (!current_object) fieldStyle.backgroundColor = 'lightgray';
```